config/default.yml

```diff
@@ -847,6 +847,17 @@ Rails/RequireDependency:
   Enabled: false
   VersionAdded: '2.10'
 
+Rails/ResponseParsedBody:
+  Description: Prefer `response.parsed_body` to `JSON.parse(response.body)`.
+  Enabled: pending
+  Safe: false
```

Comment:
```suggestion
  SafeAutoCorrect: false
```

Reply:
This change would cause an intentionally written `JSON.parse(response.body)` to be treated as an offense with safe-level `rubocop`. Is this ok?
For example, the proprietary Content-Types provided by corporate entities such as `application/vnd.github+json` do not support `response.parsed_body`, so you still `JSON.parse(response.body)` must be used in such cases.

Reply:
Yup, please go ahead. Also, that example is a very good concrete incompatibility explanation worth story adding to `@safety` description.